docs/ETL.md

```diff
@@ -98,6 +103,128 @@ graph LR;
 
 * _(Optional)_ Interface with some storage (postgres, mongo, etc.) to persist lively extracted data
 * _(Optional)_ Backtest support for polling some data between some starting and ending block heights
+* _(Optional)_ Use of an application state cache to understand which parameter sets to sequentially feed to an endpoint
+
+
+#### Example Use Case(s)
+* Polling layer-2 block data in real-time for state updates
+* Interval polling user provided chain addresses for native ETH amounts
 
 ### (TBD) Aggregator
-Aggregators are used to solve the problem where a pipe or an invariant input will require multiple data points to perform an execution sequence. Since aggregators are subscribing to more than one data stream with different output frequencies, they must employ a synchronization policy for collecting and propagating multi-data inputs within a highly asynchronous environment.
\ No newline at end of file
+**NOTE - This component type is still in-development**
+Aggregators are used to solve the problem where a pipe or an invariant input will require multiple data points to perform an execution sequence. Since aggregators are subscribing to more than one data stream with different output frequencies, they must employ a synchronization policy for collecting and propagating multi-data inputs within a highly asynchronous environment.
```

Comment:
```suggestion
Aggregators are used to solve the problem where a pipe or an invariant input will require multiple sources of data to perform an execution sequence. Since aggregators are subscribing to more than one data stream with different output frequencies, they must employ a synchronization policy for collecting and propagating multi-data inputs within a highly asynchronous environment.
```